spec/menu_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish_klass) {double(:dish_klass)}
+  let(:dish) {double(:dish)}
+  let(:dish2) {double(:dish)}
+  let(:dish_list) { [ {name: :name_arg, price: :price_arg},
+                      {name: :name_arg, price: :price_arg} ]
+                    }
+
+  subject(:menu) {described_class.new(dish_klass,dish_list)}
+
+  before do
+    allow(dish_klass).to receive(:new).with(:name_arg, :price_arg).and_return(dish,dish2)
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)